### DISCUSSION TITLE
Bar Assistant Homepage Widget

### DISCUSSION THREAD

**Discussion by cadeluca** [karlomikus/bar-assistant#340] [Show and tell]
Hi, just figured I'd share this Bar Assistant widget for [Homepage](https://github.com/gethomepage/homepage) that I'm working on!

![BAR_ASSISTANT_EX](https://github.com/user-attachments/assets/0b3862af-6c89-435d-bf43-900b7a458e14)

I recently set up Bar Assistant and love it! I self-host various other projects including Homepage as my dashboard. In that project 'service widgets' are used to hit APIs for stats. My prototype widget pulls some simple counts on total ingredients, cocktails, and what you've got on your shelf. I'm open for suggestions on what else to display; I've only just started exploring the API.

Homepage's contributor requirements do not allow feature PRs until a request upvote threshold is reached, so for now this lives in my custom image. Homepage also supports a custom api widget, which I may explore to see if I can mimic the funtionality without code changes in that project. If there is any interest I'd be happy to share code or a Gist if the custom api thought works out.

**Comment by karlomikus** [comment]
Hello,
nice, that looks great. Depending on how much code there is we could add it to the docs.
